client/src/components/NavBottom.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import colors from '../utils/colors';
+
+const Container = styled.ul`
+  display: flex;
+  height: 83px;
+  width: 100vw;
```

Comment:
this could potentially lead to some overflow if you decide to add margins at a higher level for example, just sayin'